tests/lang/de/test_tagger.py

```diff
@@ -23,15 +24,6 @@ def test_de_tagger_example(NLP):
         assert token.tag_ == expected_tag
 
 
-# This threshold is artificially low due to problems with spacy 2.1. (#3830)
-@pytest.mark.parametrize(
-    "test_file,accuracy_threshold", [("de_pud-ud-test.stts.json", 0.94)]
```

Comment:
@adrianeboyd : while we're at it, should we increase this threshold again?

Reply:
No, let's not modify any thresholds in this PR so that we can test v3.3.0 again with the exact same setup. Adjusting the thresholds can come later, especially for model size-specific ones.

Reply:
That makes sense, I just wanted to flag this as the "TODO" part was being removed ;-)

Reply:
(I'm not even sure the comment is still accurate?)